aya-log/src/lib.rs

```diff
@@ -67,12 +67,14 @@ use std::{
 
 const MAP_NAME: &str = "AYA_LOGS";
 
+pub const LEVEL: &str = "AYA_LOG_LEVEL";
```

Comment:
what's going on with these consts why are they not after all the imports?

Reply:
Seems unrelated to this change!